### ISSUE TITLE
emonavi update / emonavi 更新しました

### ISSUE THREAD

**Issue by muooon** [kozistr/pytorch_optimizer#433]
## emonavi v3.0

[emonavi](https://github.com/muooon/EmoNavi)

Thank you always for your efforts and achievements on the Pytorch-Optimizer project.

Now, we have EmoNavi v3.0.
It maintains autonomy for resuming training or in distributed learning (no need to inherit hyperparameters or synchronize for cooperation), while achieving enhanced learning and generalization through the abstraction of higher-order moments (it abstracts higher-order moments such as the rate of loss fluctuation using the difference between short- and long-term EMA to intervene in learning).
This allows EmoNavi to demonstrate its features without the shadow-system enabled and without additional tensors. As a result, it is lightweight and should be effective even in long-term or extremely large-scale training. The same applies to the variants: EmoFact, EmoLynx, EmoClan, EmoZeal, and EmoNeco. We would be grateful if you could integrate it into the Pytorch-Optimizer.

いつもお世話になっております、Pytorch-Optimizer の取り組みと実績に心から感謝しています。

さて、emonavi v3.0 になりました。
学習の再開等や分散学習等での自律性を維持(ハイパーパラメータの引き継ぎ不要、協調のための同期も不要)しつつ、高次momentの抽象化による学習強化と汎化性の獲得(短期･長期のEMAの差分で損失変動率という高次momentを抽象化し学習に介入)を達成しました。
これにより shadow-system を False のまま、追加テンソルなしで emonavi の特徴を発揮します。軽量化を果たしたので長期学習や超大規模学習でも効果的に動作すると思います。バリエーションの emofact、emolynx、emoclan、emozeal、emoneco、も同様です。お手数ですが Pytorch-Optimizer に統合してくださると嬉しいです。
